README.md

```diff
@@ -1 +1,249 @@
-# cozy-homes-back-end
\ No newline at end of file
+
+<a name="readme-top"></a>
```

Comment:
- [ ] OPTIONAL: Your backend does not have any unit tests.  I suggest you add some in order to ensure your app's integrity. 👍 

If you add this feature, don't forget to add a 'Tests' section on your readme with instructions about how to run them.

Reply:
Thank you very much for your support and your suggestions, we will add this change to future features, since it was not within the requirements we did not do unit and integration tests.